### DISCUSSION TITLE
Deploy bug

### DISCUSSION THREAD

**Discussion by Mistik** [vercel/community#3858] [Bug Report]
### Page to Investigate

Deploy

### Steps to Reproduce

Getting this error when deploying

Command failed with exit code 1: du -ms /vercel/path0/node_modules du: cannot access ‘/vercel/path0/node_modules’: No such file or directory

**Comment by binarykore** [comment]
Same.. When might this get fixed? :)

**Comment by amyegan** [comment]
@Mistik @binarykore I wasn't able to reproduce this error with my test projects. Would either of you be able to share your project or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can dig into this more?

**Reply by majidemad** [reply]
@amyegan Guess this was a transitional bug on the platform, just my assumption. We all had this momentarily. 
https://github.com/orgs/vercel/discussions/3857

**Reply by amyegan** [reply]
Thanks for letting me know
